### PR TITLE
[ACM-6859]: Removing technology preview labels for bare metal and kubevirt docs

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-6859

### CODE REVIEW

**Comment by lahinson** [comment]
@zanetworker When you can, PTAL at this PR, which removes the Technology Preview labels and language from the bare metal and KubeVirt docs. Thank you!

**Comment by zanetworker** [comment]
@lahinson lgtm, let's also get an ack from @peterclauterbach

**Comment by peterclauterbach** [comment]
LGTM

File: clusters/release_notes/whats_new.adoc [hunk 1]
```diff
@@ -24,8 +24,15 @@ Learn about what's new relating to Cluster lifecycle with {mce-short}.
...
+** xref:../hosted_control_planes/configure_hosted_bm.adoc#configuring-hosting-service-cluster-configure-bm[Configuring the hosting cluster on bare metal]
+** xref:../hosted_control_planes/managing_hosted_bm.adoc#hosted-control-planes-manage-bm[Managing hosted control plane clusters on bare metal]
+** xref:../hosted_control_planes/managing_hosted_kubevirt.adoc#hosted-control-planes-manage-kubevirt[Managing hosted control plane clusters on OpenShift Virtualization]
 
-* Hosted clusters are now automatically imported to your {mce-short} cluster. If you want to disable this feature, see xref:../hosted_control_planes/hosted_disable_auto_import.adoc#hosted-disable-auto-import[Disabling the automatic import of hosted clusters into {mce-short}].
+* Technology Preview: You can provision a hosted control plane cluster on the Amazon Web Services platform. See xref:../hosted_control_planes/configure_hosted_aws.adoc#hosting-service-cluster-configure-aws[Configuring the hosting cluster on AWS (Technology Preview)] for additional information.
+//add IBM Power/IBM z when docs are available - ACM-7407
```
Comment (oafischer) [inline clusters/release_notes/whats_new.adoc]:
```suggestion
//add IBM Power/IBM z when docs are available - ACM-7407 | LH 10/3/23
```

**Review by oafischer — APPROVED** [review]
Looks great! Just added some details to your comment

**Comment by lahinson** [comment]
Thanks @oafischer! I'll update that comment - good catch.

**Pull request merged**